File: ProjectPendragon/src/app/models/character/attributes.ts
export class Attributes {
    id: string;
    siz: number;
    dex: number;
    str: number;
    con: number;
    app: number;

    constructor() {
        this.id = crypto.randomUUID();
        this.siz = 5;
        this.dex = 5;
        this.str = 5;
        this.con = 5
        this.app = 5;
    }
}

File: ProjectPendragonBackend/Models/Attributes.cs
namespace ProjectPendragonBackend.Models
{
    public class Attributes
    {
        public Guid Id { get; set; }
        public Guid CharacterId { get; set; }
        public Character Character { get; set; }
        public int Siz { get; set; }
        public int Dex { get; set; }
        public int Str { get; set; }
        public int Con { get; set; }
        public int App { get; set; }
    }
}